compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -190,10 +190,17 @@ OMR::ARM64::CodeGenerator::initialize()
          cg->setSupportsArrayCmpLen();
          }
       }
-      if (!comp->getOption(TR_DisableArraySetOpts))
-         {
-         cg->setSupportsArraySet();
-         }
+
+   if (!comp->getOption(TR_DisableArraySetOpts))
+      {
+      cg->setSupportsArraySet();
+      }
+
+   static bool disableTRTO255 = (feGetEnv("TR_disableTRTO255") != NULL);
+   if (!disableTRTO255)
+      {
+      cg->setSupportsArrayTranslateTRTO255();
+      }
```

Comment:
I don't think support for this opcode should be enabled in OMR when there isn't an implementation for it in OMR (the helper implementation is missing).  In the absence of that, this logic should be enabled in the downstream projects that do provide the helper.

Reply:
Both p and x platforms do the same: Implement the helper functions in the downstream project, and enable the feature in OMR.
On the other hand, z inlines the vectorized code instead of calling helper functions.

I can move the implementation of the AArch64 helper functions to OMR.  In that case, maybe we want to move the helper functions for p and x to OMR for consistency.

[p]
- https://github.com/eclipse-openj9/openj9/blob/master/runtime/compiler/p/runtime/J9PPCArrayTranslate.spp
- https://github.com/eclipse/omr/blob/205c540402c01986f433a34412f9d7fcfe61d90f/compiler/p/codegen/OMRCodeGenerator.cpp#L250-L262

[x]
- https://github.com/eclipse-openj9/openj9/blob/master/runtime/compiler/x/runtime/X86ArrayTranslate.nasm
- https://github.com/eclipse/omr/blob/205c540402c01986f433a34412f9d7fcfe61d90f/compiler/x/codegen/OMRCodeGenerator.cpp#L424-L447